tensorflow/python/kernel_tests/rnn_test.py

```diff
@@ -574,59 +574,16 @@ def _testDynamicEquivalentToStaticRNN(self, use_gpu, use_sequence_length):
       sequence_length = None
 
     ########### Step 1: Run static graph and generate readouts
-    with self.test_session(use_gpu=use_gpu, graph=tf.Graph()) as sess:
```

Comment:
those test code refactored into run_static_rnn so that it can be shared.

--
William Chan
Carnegie Mellon University
(650) 450-9455
williamchan.ca

On Wed, May 11, 2016 at 10:02 PM, ebrevdo <notifications@github.com> wrote:

> In tensorflow/python/kernel_tests/rnn_test.py
> <https://github.com/tensorflow/tensorflow/pull/2002#discussion_r62965383>:
>
> > @@ -574,59 +574,16 @@ def _testDynamicEquivalentToStaticRNN(self, use_gpu, use_sequence_length):
> >        sequence_length = None
> >
> >      ########### Step 1: Run static graph and generate readouts
> > -    with self.test_session(use_gpu=use_gpu, graph=tf.Graph()) as sess:
>
> did this get moved elsewhere?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2002/files/8b6f595036ec7829cb78f985883f3e579254dc69#r62965383>
>